pillar/shibboleth/odlvideo.sls

```diff
@@ -3,7 +3,5 @@
 
 nginx-shibboleth:
   secrets:
-    key: |
-      {{ salt.vault.read('secret-odl-video/{env}/shibboleth/sp-key'.format(env=ENVIRONMENT)).data.value|replace('\\n', '\n')|indent(6) }}
-    cert: |
-      {{ salt.vault.read('secret-odl-video/{env}/shibboleth/sp-cert'.format(env=ENVIRONMENT)).data.value|replace('\\n', '\n')|indent(6) }}
+    key: __vault__::secret-odl-video/{{ ENVIRONMENT }}/shibboleth/sp-key>data>value
```

Comment:
Is this gonna work since the cert stored in vault actually does have `\n` in it or should we just that in vault?

Reply:
That's unclear so it's something that we should test once we merge this